supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -332,6 +333,8 @@ class AwsBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
 
   // Primary entry point for cromwell to actually run something
   override def executeAsync(): Future[ExecutionHandle] = {
+    implicit val timer: Timer[IO] = cats.effect.IO.timer(ExecutionContext.fromExecutor(Executors.newSingleThreadScheduledExecutor))
```

Comment:
@danbills I had to upgrade to fs2 1.0.0 to make use of `IOApp`. In 1.0 they removed the Scheduler as cats effect Timer does pretty much the same thing already: https://github.com/functional-streams-for-scala/fs2/blob/topic/cats-effect-0.10/docs/migration-guide-1.0.md#timer--scheduler

I replaced the Scheduler bits with Timer bits but if you could take a look that'd be great :)

Reply:
TIL.  looking now

Reply:
lgtm